config/prod/tesla-analysis.yaml

```diff
@@ -0,0 +1,23 @@
+# Provision an EC2 instance
+template_path: templates/managed-ec2-v2.yaml
+stack_name: tesla-analysis
```

Comment:
would you mind adding `ec2` the this name?  and also change the filename to match?  something like `tesla-analysis-ec2` would work.